d3ledger/back-office-admin
# D3 | Reports

# Description
https://soramitsu.atlassian.net/browse/D3-1065
https://soramitsu.atlassian.net/browse/D3-1044

# Works done
- [x] Custody billing report
- [x] Account registration billing report
- [x] Transfer billing report
- [x] Exchange billing report

# How to test 
0. run back-office-admin, back-office, notary
1. Set fee
2. Make all basic actions 
3. Check report pages
